pkg/stub/handler.go

```diff
@@ -1624,6 +1629,24 @@ func (g *defaultConfigMapClientWrapper) Delete(namespace, name string) error {
 	return g.h.coreclient.ConfigMaps(namespace).Delete(name, &metav1.DeleteOptions{})
 }
 
+type defaultConfigClientWrapper struct {
+	h *Handler
+}
+
+func (g *defaultConfigClientWrapper) Get(name string) (*configv1.ClusterOperator, error) {
+	return g.h.configclient.ClusterOperators().Get(name, metav1.GetOptions{})
+}
+
+func (g *defaultConfigClientWrapper) Update(state *configv1.ClusterOperator) error {
+	_, err := g.h.configclient.ClusterOperators().UpdateStatus(state)
+	return err
+}
+
+func (g *defaultConfigClientWrapper) Create(state *configv1.ClusterOperator) error {
+	_, err := g.h.configclient.ClusterOperators().Create(state)
+	return err
+}
```

Comment:
seems like all of this should be moved up to where OperatorSDKHandler is defined.  (or that interface definition should move down here).

Reply:
I'll move the interface def from operatorstatus.go to handler.go

Reply:
check that ... the other direction